src/sdgio-edit.owl

```diff
@@ -2727,6 +2749,22 @@ AnnotationAssertion(rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020235> "Mult
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020235> "improvement of natural resources and ecosystem services system level outcome"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020232> <http://purl.unep.org/sdg/SDGIO_00020235>)
 
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020238> (Increased resilience of the poor to climate change and other shocks)
```

Comment:
Remove all capital letters that are not part of proper nouns.

Reply:
Done. 
Just to understand, several names of instances begin with capital letters even if the words are not proper nouns. Is there a general rule? (I got a screenshot that I can share by email. Apparently, I cannot attach files here.)

Reply:
That shouldn't be the case, could you add a few examples here?
In some cases (e.g. targets, indicators), the instance labels are essentially sentences, which follow sentence grammar.